scripts/convert_svd_to_diffusers.py

```diff
@@ -1,3 +1,14 @@
+import argparse
```

Comment:
Can we please keep this untouched and create a fresh script, instead?

Reply:
Sure, I don't mind. But these changes are not MotionCtrl specific. With these changes, I simply expose a CLI for converting SVD based checkpoints easily.

Reply:
Also, some of these changes are inspired from #6843, but the script there was incomplete and only switched the unet. This aims to be a full svd checkpoint conversion (but it still needs the clip vision encoder conversion instead of the hardcode).

Reply:
Then, ideally, this should be a separate PR fixing the SVD conversion script.

Reply:
Oh okay. I'll unstage from here and add along with from_single_support separately.

Reply:
Or it might be best to wait for https://github.com/huggingface/diffusers/pull/6843 to get merged and then proceed here?

Reply:
Yep, makes sense. @Yanting-K If you find time, could you adapt the full conversion from here instead of just the unet?

Reply:
sayakpaul is update this script!

Reply:
@Yanting-K feel free to do that in https://github.com/huggingface/diffusers/pull/6843 :-) I just provided you with an example, feel free to repurpose it.

Reply:
okey, I'll check it.